make-uniform/transitland_feed/standard_route_names.csv

```diff
@@ -0,0 +1,516 @@
+"","survey","entry","base_name","operator"
```

Comment:
No disagreement for the big operators (e.g. SF Muni or BART), but do you have any preferences for smaller operators that won't have their own survey (e.g. SF Bay Harbor Ferry, Blue & Gold)?

Reply:
Could we revisit the canonical names? I'd like to suggest that they should meet a uniform string format (e.g. "CAPS", 4 letters, no spaces, or "lower_underscore", or "lower space"). It doesn't make all that much difference in the code, but I wanted to raise the issue since we started with a mandate to meet a style guide. And, while hadley's style guide is silent on coded variable values, we are choosing these names rather than having them thrust upon us by the surveyor.

Reply:
There are lots of records that won't have a GTFS feed (e.g. SamTrans, the many company based shuttles, and the Ferries). We can crosswalk into something static later, but for now I want to try just crosswalking between the surveys. 

Or, for operators that don't have gtfs feeds, I can assign them a generic operator ("Missing")?